src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -256,9 +260,9 @@ public TransactionEnvelope toEnvelopeXdr() {
      * who will use a sequence number. When build() is called, the account object's sequence number
      * will be incremented.
      */
-    public Builder(TransactionBuilderAccount sourceAccount, Network network) {
-      checkNotNull(sourceAccount, "sourceAccount cannot be null");
-      mSourceAccount = sourceAccount;
+    public Builder(AccountConverter accountConverter, TransactionBuilderAccount sourceAccount, Network network) {
+      mAccountConverter = checkNotNull(accountConverter, "accountConverter cannot be null");
```

Comment:
Maybe a default to non-muxed (since we want opt-in) would be more convenient here?